griptape/tools/inpainting_image_generation_client/tool.py

```diff
@@ -31,17 +31,11 @@ class InpaintingImageGenerationClient(BlobArtifactFileOutputMixin, BaseTool):
 
     @activity(
         config={
-            "description": "Can be used to modify an image within a specified mask area using image and mask files.",
+            "description": "Modifies an image within a specified mask area using image and mask files.",
             "schema": Schema(
                 {
-                    Literal(
-                        "prompts",
-                        description="A detailed list of features and descriptions to include in the generated image.",
-                    ): list[str],
-                    Literal(
-                        "negative_prompts",
-                        description="A detailed list of features and descriptions to avoid in the generated image.",
-                    ): list[str],
+                    Literal("prompt", description=BaseImageGenerationClient.PROMPT_DESCRIPTION): str,
+                    Literal("negative_prompt", description=BaseImageGenerationClient.NEGATIVE_PROMPT_DESCRIPTION): str,
```

Comment:
Are you making it singular so that the LLM will rely on parallel actions if needs to generate multiple images?

Reply:
This doesn't change how many images are generated per invocation, the behavior remains to generate one image per activity call. The core of this change is that asking an Agent to provide one continuous [negative] prompt string in the schema produces better adherence to the schema than asking for a JSON list of "prompts" concatenated into a string by the tool.